elfpy/hyperdrive_interface/hyperdrive_interface.py

```diff
@@ -203,6 +204,8 @@ def get_wallet_info(
         token_maturity_time = transaction.event_maturity_time
         if wallet_addr is None:
             continue
+
```

Comment:
no newlines pls

Reply:
This is breaking up logic and I'm using it sparingly

Reply:
I'll remove for now, but I'll fight you tooth and nail on this. For me, breaking up logic with new lines makes the code much easier to read (comments doesn't do the same for me)